data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/ReceiverVerticle.java

```diff
@@ -160,6 +153,8 @@ public void start(final Promise<Void> startPromise) {
             }
         }
 
+        authVerifier.start(vertx);
```

Comment:
I was expecting that this is somehow returning an error / Future to report when discovery fails, how are we handling that?